### ISSUE TITLE
Clear Inventory no longer handles ** (double star) properly.

### ISSUE THREAD

**Issue by awgybop1** [EssentialsX/Essentials#3638]
### Information

**Full output of `/ess version`**:

```
Server version: 1.8.8-R0.1-SNAPSHOT git-Spigot-21fe707-e1ebe52 (MC: 1.8.8)
29.08 18:32:43 [Server] INFO EssentialsX version: 2.18.1.0
29.08 18:32:43 [Server] INFO PlaceholderAPI version: 2.10.6
29.08 18:32:43 [Server] INFO GroupManager version: Pre2.14.1.3 FINAL (Phoenix)
29.08 18:32:43 [Server] INFO Vault version: 1.5.6-b49
29.08 18:32:43 [Server] INFO Citizens version: 2.0.16-SNAPSHOT (build 1267)
29.08 18:32:43 [Server] INFO EssentialsXChat version: 2.18.1.0
29.08 18:32:43 [Server] INFO EssentialsXSpawn version: 2.18.1.0
29.08 18:32:43 [Server] INFO ChestShop version: 3.8.12
```

### Details

**Description**  
/ci ** clears everyone's inventory.

**Steps to reproduce**  
Use the command /clear **, /clearinventory **, or /ci **

**Expected behavior**  
/ci ** is supposed to clear only my Inventory and Armor Slots.

**Comment by Chew** [comment]
I believe just /ci clears all your inventory, doing ** to target everyone seems intentional.

**Comment by awgybop1** [comment]
This never happened before, in previous versions the double star always
only targeted me and was used to clear both your inventory and armor slots
- something just /ci won't do.

**Comment by JRoy** [comment]
Wow I really messed up this command, fix soon.

**Comment by JasonHorkles** [comment]
The command is `/ci <player> **`, so it seems like it's doing what it should

**Comment by awgybop1** [comment]
Yes, but the problem is that `/ci **` should do like all commands and target myself (ex. /god, /fly, etc.) unless a player is specified.
`/ci *` clears all inventories, not `/ci **`

And thank you @JRoy !

**Comment by JasonHorkles** [comment]
I feel like that would cause more confusion than anything, since ** is already inconsistent enough with what it does

**Comment by awgybop1** [comment]
Well, that's the way it has been for years now and I've grown accustom to it.
I do get your point though, especially when commands like `/eco set ** 0` would clear EVERYONE'S balance on the server, one might think `/ci **` clears everyone's inventory (offline and online).

Yet commands like `/kit <kit> *` and `/kit <kit> **` do absolutely nothing - really strange.

**Comment by mdcfe** [comment]
This has been fixed as of f89840545d025d6eb71fccd513b646fa6a3ce35f - you can download the latest version of EssentialsX from the [website](https://essentialsx.net/downloads.html).
